week2/velog/velogTag.js

```diff
@@ -0,0 +1,56 @@
+//로컬스토리지에서 태그 가져와서 배열 형태로 바꾸기
+let tagList = JSON.parse(localStorage.getItem("tag"));
+if (tagList === null) {
+  tagList = [];
+}
+
+const tagIsOkay = (tag) => {
+  if (tag == undefined || tag == null || tag.length <= 0) {
+    return false;
+  } else if (tagList.includes(tag) == true) {
+    alert("중복된 태그는 추가할 수 없습니다!");
+    return false;
+  }
+  return true;
+};
+
+const addTag = (e) => {
+  if (e.keyCode === 13 && tagIsOkay(e.target.value)) {
```

Comment:
```suggestion
  if (e.key === 'Enter' && tagIsOkay(e.target.value)) {
```

e.keycode로 키입력을 감지하는 방식은 더이상 사용이 권장되지 않는방식(deprecated)라서,  위같은 표기법을 사용하는게 알맞다고 합니다 !!